src/data/faq.json

```diff
@@ -1904,11 +1904,12 @@
                         ]
                     },
                     {
-                        "title": "I received the message that I had a low risk encounter. The next day it was gone again. Shouldn't this be deleted after 14 days?",
+                        "title": "I received the message that I had a risk encounter. The next day it was gone again. Shouldn't this be deleted after 14 days?",
                         "anchor": "encounter_deletion",
                         "active": false,
                         "textblock": [
-                            "As soon as the app downloads the current diagnostic keys (positive IDs) from the server, a check of the contact log is performed. For each of the last 14 days for which diagnostic keys are available, a separate contact log check is performed. This means that the app compares the encounters of the last 14 days with the positive IDs to check whether there has been a contact with a person who tested positive. If the message about a low-risk encounter disappears after one day, this means that the corresponding encounter was more than 14 days ago in this case."
+                            "As soon as the app downloads the current diagnosis keys (positive IDs) from the server, a check of the contact log is performed. For each of the last 14 days for which diagnosis keys are available, a separate contact log check is performed. This means that the app compares the encounters of the last 14 days with the positive IDs to check whether there has been a contact with a person who tested positive. If the message about a risk encounter disappears after one day, this means that the corresponding encounter was more than 14 days ago in this case.",
+                            "You can find more information on when your risk encounter happened in the contact journal. Further information on this feature are available in the blog post <a href='../blog/2021-02-10-corona-warn-app-version-1-12' target='_blank'>Corona-Warn-App version 1.12 comes with two new features</a> in the paragraph 'Contact journal with exposure history'."

```

Comment:
"are" => "is" 
in
"Further information on this feature are ..."

"information" is an uncountable noun always used in the singular, see https://dictionary.cambridge.org/grammar/british-grammar/information

Reply:
```suggestion
                            "You can find more information on when your risk encounter happened in the contact journal. Further information on this feature is available in the blog post <a href='../blog/2021-02-10-corona-warn-app-version-1-12' target='_blank'>Corona-Warn-App version 1.12 comes with two new features</a> in the paragraph 'Contact journal with exposure history'."
```